_chapters/js/style.md

```diff
@@ -1,6 +1,45 @@
 ---
-title: Style
+title: Style / Linting
 slug: js-style
 ---
-We currently recommend adhering to the [AirBnB](https://github.com/airbnb/javascript) JavaScript style guide. 
-Your team may, of course, choose a different style for your project if you all agree that you have reason to do so.
+We recommend adhering to the [Airbnb JavaScript style guide](https://github.com/airbnb/javascript).
```

Comment:
But "currently" is kind of implied, otherwise we'd be qualifying nearly all the statements in the guide.